ovh/resource_ovh_cloud_kubernetes_cluster_test.go

```diff
@@ -0,0 +1,76 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var testAccPublicCloudProjectKubeConfig = fmt.Sprintf(`
+resource "ovh_cloud_project_kube" "cluster" {
+	project_id  = "%s"
+  	name = "my cluster for acceptance tests"
+	region = "GRA5"
+	version = "1.15"
+}
+`, os.Getenv("OVH_PUBLIC_CLOUD"))
+
```

Comment:
a sweeper function has to be implemented

example here
https://github.com/ovh/terraform-provider-ovh/blob/master/ovh/resource_ovh_iploadbalancing_tcp_farm_test.go#L56